mlflow/models/cli.py

```diff
@@ -270,3 +272,51 @@ def build_docker(**kwargs):
     'python_function' flavor.
     """
     python_api.build_docker(**kwargs)
+
+
+@commands.command("update-pip-requirements")
+@cli_args.MODEL_URI
+@click.argument("operation", type=click.Choice(["add", "remove"]))
+@click.argument("requirement_string", type=str)
+def update_pip_requirements(model_uri, operation, requirement_string):
+    """
+    Add or remove requirements from a model's conda.yaml and requirements.txt files.
+    If using a remote tracking server, please make sure to set the MLFLOW_TRACKING_URI
+    environment variable to the URL of the desired server.
+
+    REQUIREMENT_STRING is a comma-separated string of pip requirements,
+    e.g. "pandas==1.0.0, scikit-learn".
+
+    Sample usage:
+
+    .. code::
+
+        # Add requirements using the model's "runs:/" URI
+
+        mlflow models update-pip-requirements -m runs:/<run_id>/<model_path> \\
+            add "pandas==1.0.0, scikit-learn"
+
+        # Remove requirements from a local model
+
+        mlflow models update-pip-requirements -m /path/to/local/model \\
+            remove "torchvision, pydantic"
+
+    Note that model registry URIs (i.e. URIs in the form ``models:/``) are not
+    supported, as artifacts in the model registry are intended to be read-only.
+    Editing requirements is read-only artifact repositories is also not supported.
+
+    If adding requirements, the function will overwrite any existing requirements
+    that overlap, or else append the new requirements to the existing list.
+
+    If removing requirements, the function will ignore any version specifiers,
+    and remove all the specified package names. Any requirements that are not
+    found in the existing files will be ignored.
+    """
```

Comment:
/ai what verb would you use for the add operation?

Reply:
/ai how would you describe `'append' if the requirement doesn't already exist, and 'overwrite' if the requirement does already exist.` with a single verb?

Reply:
Got it :)

Reply:
"upsert" came across my mind but I'm fine with `add`.